hubblestack/extmods/hubble_mods/misc.py

```diff
@@ -0,0 +1,1096 @@
+        }
+    }    
+
+    ret_status, ret_val = hubblestack.extmods.module_runner.comparator.run(
+        block_id, comparator_args, path_details.get('mode'))
+    return True if ret_status else path_details.get('mode')
+    
+def _check_core_dumps(block_id, block_dict, extra_args):
+    """
+    Ensure core dumps are restricted
+    """
+    hard_core_dump_value = _execute_shell_command("grep -R -E \"hard +core\" /etc/security/limits.conf /etc/security/limits.d/ | awk '{print $4}'", python_shell=True).strip()
+    hard_core_dump_value = hard_core_dump_value.split('\n') if hard_core_dump_value != "" else []
+    if '0' in hard_core_dump_value:
+        return True
+
+    if hard_core_dump_value is None or hard_core_dump_value == [] or hard_core_dump_value == "":
+        return "'hard core' not found in any file"
+
+    return str(hard_core_dump_value)
+
+def _check_service_status(block_id, block_dict, extra_args=None):
+    """
+    Ensure that the given service is in the required state. Return False if it is not in desired state
+    Return True otherwise
+    state can be enabled or disabled.
+    """
+    service_name = runner_utils.get_param_for_module(block_id, block_dict, 'service_name')
+    state = runner_utils.get_param_for_module(block_id, block_dict, 'state')
+
+    all_services = __salt__['cmd.run']('systemctl list-unit-files')
+    if re.search(service_name, all_services, re.M):
+        output = __salt__['cmd.retcode']('systemctl is-enabled ' + service_name, ignore_retcode=True)
+        if (state == "disabled" and str(output) == "1") or (state == "enabled" and str(output) == "0"):
+            return True
+        else:
+            return __salt__['cmd.run_stdout']('systemctl is-enabled ' + service_name, ignore_retcode=True)
+    else:
+        if state == "disabled":
+            return True
+        else:
+            return 'Looks like ' + service_name + ' does not exists. Please check.'
+
+def _check_ssh_timeout_config(block_id, block_dict, extra_args):
+    """
+    Ensure SSH Idle Timeout Interval is configured
+    """
+
+    client_alive_interval = _execute_shell_command("grep \"^ClientAliveInterval\" /etc/ssh/sshd_config | awk '{print $NF}'", python_shell=True).strip()
+    if client_alive_interval != '' and int(client_alive_interval) <= 300:
+        client_alive_count_max = _execute_shell_command("grep \"^ClientAliveCountMax\" /etc/ssh/sshd_config | awk '{print $NF}'", python_shell=True).strip()
+        if client_alive_count_max != '' and int(client_alive_count_max) <= 3:
+            return True
+        else:
+            return "ClientAliveCountMax value should be less than equal to 3"
+    else:
+        return "ClientAliveInterval value should be less than equal to 300"
+
+def _check_unowned_files(block_id, block_dict, extra_args):
+    """
+    Ensure no unowned files or directories exist
+    """
+    raise CommandExecutionError('Module disabled due to performance concerns')
```

Comment:
No need of any implementation if this module is disabled. Infact we can remove this functionality altogether from this module.